checks/__init__.py

```diff
@@ -889,7 +889,7 @@ def normalize(self, metric, prefix=None, fix_case=False):
             if prefix is not None:
                 prefix = self.convert_to_underscore_separated(prefix)
         else:
-            name = re.sub(r"[,\+\*\-/()\[\]{}\s]", "_", metric_name)
+            name = re.sub(r"[,\@\+\*\-/()\[\]{}\s]", "_", metric_name)
```

Comment:
Could you also check if we should fix, `normalize` for the main Check class (https://github.com/DataDog/dd-agent/blob/ea4b687c87719e7bb5d8992df059d414ad6d2a22/checks/__init__.py#L92) here as well? In reality most checks should inherit from `AgentCheck` but it might make sense for legacy custom checks.

Reply:
I don't think it needs to be updated there since its a deprecated class, but it doesn't hurt to add it :)

Reply:
I was asking in case customers had implemented custom checks based on the deprecated class :)

Reply:
Ahh, that makes sense. Good call, thanks!